nifi-commons/nifi-flowfile-packager/src/main/java/org/apache/nifi/util/FlowFilePackagerV3.java

```diff
@@ -37,7 +37,7 @@ public void packageFlowFile(final InputStream in, final OutputStream out, final
             writeFieldLength(out, attributes.size()); //write out the number of attributes
             for (final Map.Entry<String, String> entry : attributes.entrySet()) { //write out each attribute key/value pair
                 writeString(entry.getKey(), out);
-                writeString(entry.getValue(), out);
+                writeString(entry.getValue() == null ? "" : entry.getValue(), out);
```

Comment:
@mosermw we gotta dump this line based on last convo.  i'll do it locally prior to merge.

Reply:
err this is the actual value.  i dont recall whether we enforce that or not